libnmstate/__init__.py

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (c) 2018-2019 Red Hat, Inc.
```

Comment:
Please update the commit message, it is not a method anymore.

Reply:
Done, thanks!